[Search Profiler] Improve text color contrast ratio

ElenaStoeva opened the pull request:
Closes https://github.com/elastic/kibana/issues/154424

## Summary

This PR improves the color contrast ratio of the text in Search profiler by replacing the text with `EuiEmptyPrompt` in both the empty state component and the loading state component.

<img width="1184" alt="Screenshot 2023-10-30 at 10 58 42" src="https://github.com/elastic/kibana/assets/59341489/b897cde4-2063-4de6-812c-b38a5be5a725">
<img width="1184" alt="Screenshot 2023-10-30 at 10 58 28" src="https://github.com/elastic/kibana/assets/59341489/c384128a-2c79-4443-a2e0-c690eb2043c4">
<img width="1184" alt="Screenshot 2023-10-30 at 10 58 35" src="https://github.com/elastic/kibana/assets/59341489/9e31910b-69ad-480e-b499-64ffa1355b5c">


<img width="1152" alt="Screenshot 2023-10-30 at 11 05 10" src="https://github.com/elastic/kibana/assets/59341489/f9cbd85b-01ba-4432-9768-8d6635045937">


Note: https://github.com/elastic/kibana/issues/154424 also mentions that the tabs in Search profiler have low contrast ratio and I addressed this in https://github.com/elastic/kibana/issues/154424#issuecomment-1777693100.

MichaelMarcialis reviewed (approved):
Code LGTM. Approving.

As a side note (either for this or a future PR), it may make sense to use an `EuiEmptyPrompt` component here, if only for the sake of typographic consistency with other similar messages across Kibana.

ElenaStoeva commented:
@elasticmachine merge upstream

ElenaStoeva commented:
> Code LGTM. Approving.
> 
> As a side note (either for this or a future PR), it may make sense to use an `EuiEmptyPrompt` component here, if only for the sake of typographic consistency with other similar messages across Kibana.

Thanks for the review and for the suggestion @MichaelMarcialis! I added the `EuiEmptyPrompt` component and updated the screenshots in the PR. Let me know if you have any additional feedback.

sabarasaba reviewed (approved):
Thanks for patching this up @ElenaStoeva! changes lgtm
